src/Microsoft.ML.Data/Evaluators/Metrics/ClusteringMetrics.cs

```diff
@@ -26,30 +26,30 @@ public sealed class ClusteringMetrics
         /// Note however, that this metric will only decrease if the number of clusters is increased,
         /// and in the extreme case (where each distinct example is its own cluster) it will be equal to zero.
         /// </summary>
-        public double AvgMinScore { get; }
+        public double AverageMinimumScore { get; }
 
         /// <summary>
         /// <a href="https://en.wikipedia.org/wiki/Davies%E2%80%93Bouldin_index">Davies-Bouldin Index</a>
         /// DBI is a measure of the how much scatter is in the cluster and the cluster separation.
         /// </summary>
```

Comment:
[https://scikit-learn.org/stable/modules/generated/sklearn.metrics.davies_bouldin_score.html#sklearn.metrics.davies_bouldin_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.davies_bouldin_score.html#sklearn.metrics.davies_bouldin_score) I would suggest to be inspired by that documentation.
I would put links to external web sites in `<remarks>`
Summary should have short explanation of that it's about.